src/content/contribution_ways.js

```diff
@@ -0,0 +1,81 @@
+const contributionWays = {
+  title: 'WAYS TO CONTRIBUTE',
+  par:
+    'Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptates repellendus delectus labore, rerum quisquam magnam excepturi impedit maxime mollitia perspiciatis.',
+  ways: [
+    {
+      title: 'CODE',
+      image: require('../assets/contribution/code.png'),
+      channels: [
+        {
+          name: 'lorem',
+          url: 'https://anitab-org.zulipchat.com/#narrow/stream/212722-general',
+        },
+        {
+          name: 'lorem',
+          url: 'https://anitab-org.zulipchat.com/#narrow/stream/212722-general',
+        },
+      ],
+      content: [
+        {
+          par:
+            'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Similique, incidunt veniam dolores id asperiores ad, rerum pariatur eligendi doloremque mollitia quas quis! Reiciendis maiores ab ullam accusantium consequatur numquam modi!',
```

Comment:
No lorem impsum will be merged

Reply:
https://github.com/anitab-org/anitab-org.github.io/pull/181 it seems this PR is about content.